src/app/core/containers/app.component.scss

```diff
@@ -20,7 +20,7 @@
 ion-app.md,
 ion-content {
   background-color: $cl-secondary;
-  //background-image: url('../assets/imgs/bg-mobile.png');
+  background-image: url('../assets/imgs/bg-mobile.png');
```

Comment:
@nivemaham I've just changed the bg image to the color of $cl-secondary because changing the $background-color variable was causing bugs in `range-input`